build/cli/src/lib.rs

```diff
@@ -836,6 +836,27 @@ pub async fn main_internal(config: Option<enso_build::config::Config>) -> Result
                 .run_ok()
                 .await?;
 
+            let build_json_exists = ctx.repo_root.join("app/ide-desktop/build.json").exists();
+            let ide_artifacts_exist = ctx.repo_root.join("target/ensogl-pack/linked-dist").exists();
+            if !build_json_exists || !ide_artifacts_exist {
+                let ide_cli = Cli::parse_from([
+                    "./run",
+                    "ide",
+                    "build",
+                    "--skip-version-check",
+                    "--skip-wasm-opt",
+                    "--wasm-profile=dev",
+                    "--backend-source",
+                    "release",
+                    "--backend-release",
+                    "nightly",
+                ]);
+                if let Target::Ide(ide) = ide_cli.target {
+                    ctx.handle_ide(ide).await?
+                } else {
+                    unreachable!("This command line is hard-coded to always use the IDE target.")
+                }
```

Comment:
Please talk with @mwu-tow about proper implementation:
1. I don't think that hardcoding flags like that is a good idea.
2. I'm not sure whether this will not drastically increase the CI time.

Anyway, I believe that @mwu-tow is the best person to chat with and discuss the proper solution here.

Reply:
1. definitely agree, will do
2. the idea was it'd only have to run on clean runners, so 0 minutes extra most of the time, 7 minutes extra the rest of the time. i didn't actually test that though, rerunning lint CI might have been a good idea